client/src/App.js

```diff
@@ -2,6 +2,7 @@ import { createContext, useEffect, useState } from "react";
 import { Route, Routes } from "react-router-dom";
 import Home from "./pages/Home";
 import Header from "./components/Header";
+import AboutUs from "./components/AboutUs";
```

Comment:
Let's rename all of these to About because it will be About the aims/features of the Project and not Us the team.

```suggestion
import About from "./components/About";
```

Reply:
<img width="1440" alt="Screenshot 2023-11-14 at 19 18 15" src="https://github.com/fazbazjaz/star/assets/113185323/a9b35869-4412-4f2f-b291-57657628d0dd">